docs/interfaces/index.md

```diff
@@ -96,38 +96,40 @@ For more information contact `@zanellia`.
     :private-members:
     :undoc-members:
 ```
+
 ``` eval_rst
-.. automodule:: acados_template.AcadosModel
+.. automodule:: acados_template.acados_ocp
```

Comment:
is this swapped with acados_model?

Reply:
Yes, since `AcadosOcp` is the struct to define the ocp and thus most interesting for the user, I thought it is most convenient to start with that and then show how the subclasses of `AcadosOcp` look like.